packages/card/Card.tsx

```diff
@@ -1,28 +1,25 @@
 import { h, Component } from 'skatejs';
 import styles from './Card.scss';
+import { shadyCssStyles } from '../_helpers';
 
-export class Card extends Component<void> {
-  static get is() { return 'bl-card'; }
+export type CardProps = Props;
```

Comment:
even that it doesn't do anything we should be probably consistent with rest of components so `Props & Events`